src/diagram.js

```diff
@@ -87,7 +87,11 @@ export function toTeX(diagram) {
 
                         edge.bend > 0 ? `bend left=${edge.bend}`.replace('=30', '')
                         : edge.bend < 0 ? `bend right=${-edge.bend}`.replace('=30', '')
-                        : null
+                        : null,
+
+                        edge.shift < 0 ? `shift left=${-edge.shift}`.replace('=1', '')
+                        : edge.shift > 1 ? `shift right=${edge.shift}`.replace('=1', '')
```

Comment:
This will turn `shift left=10` into `shift left0`... Maybe use `.replace(/=1$/, '')`. Can you also please fix the `bend left/right` code?